### ISSUE TITLE
Don't require tty for sudo

### ISSUE THREAD

**Issue by Kami** [StackStorm/st2-packages#264]
AIO installer commented out `Defaults +requiretty` line in the `/etc/sudoers` file, but the packages installation instructions and current install scripts don't do that. This will cause issues for the users (`error: sudo: sorry, you must have a tty to run sudo`).

Here are the corresponding lines from workroom.

```ruby
  file_line { 'disable_tty':
    path  => '/etc/sudoers',
    match => '^Defaults\s+requiretty',
    line  => '## Defaults    requiretty',
  }
```

We need to do the same for new packages (update scripts and docs).

**Comment by Kami** [comment]
It turned out we already kinda did this, but there were two problems with existing approach:

1. We didn't do it Ubuntu
2. We used a strict regex so it didn't match line with multiple spaces

**Comment by Kami** [comment]
Resolves as part of #264 and https://github.com/StackStorm/st2docs/pull/118.
